content/departments/product-engineering/engineering/cloud/delivery/deployment/helm.md

```diff
@@ -48,3 +48,117 @@ In the event you need assistance from Delivery, please gather the following info
 
 [deployments]: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/
 [statefulsets]: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
+
+## Helm GA Q&A
+
+#### What does GA mean here?
+
+GA (General Availability) here means that we’re offering it as a supported deployment option.
+
+But more than that, on the date of the GA, we’re going to be updating the various places that talk about Kubernetes in the docs to recommend Helm as the method for using Kubernetes.
+
+#### What is Helm?
+
+[Helm](https://helm.sh/) is a deployment and packaging tool for Kubernetes applications. It uses Go templating to allow end-users to customize their deployments with a config (“values”) file.
+
+#### Where can you find the customer Helm docs?
+
+[Here!](https://docs.sourcegraph.com/admin/install/kubernetes/helm)
+
+#### Why is Helm a better option for running Sourcegraph?
+
+Our Helm chart has a lot of sensible defaults baked into the [values.yaml](https://github.com/sourcegraph/deploy-sourcegraph-helm/blob/main/charts/sourcegraph/values.yaml). For changes to these defaults, we recommend using an override file that always lives outside the chart repo to make changes.
+
+Not only is this much better for deployment as it’s much simpler than forking the repo and maintaining the changes yourself, it also ensures that you never have to deal with merge conflicts during upgrades.
+
+#### Why are you adding it as a deployment method?
+
+- Customers have requested it
+- Helm is widely adopted and can be considered an industry standard
+- It saves customers from maintaining a fork of the deploy-sourcegraph repo and managing git conflicts when merging in the latest version.
+- Simplifies previously complex customizations for customers. For example, when using the existing [kustomize namespaced overlay](https://sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph/-/tree/overlays/namespaced) to deploy Sourcegraph into a custom namespace, users need to set the namespace in multiple files. With helm, these files pull the correct namespace value without manual input.
+
+#### How should we position Helm to customers?
+
+We recommend Helm as _**the**_ best method for deploying Sourcegraph.
```

Comment:
I'm slightly hesitant about this - I know that's our end-state but I'm not sure we want to start pushing customers hard towards Helm yet since we didn't have a ton of beta adoption. What do you think?